openage/convert/service/debug_info.py

```diff
@@ -677,3 +678,34 @@ def debug_media_cache(
 
     with logfile.open("w") as log:
         log.write(logtext)
+
+
+def debug_execution_time(debugdir: Directory, loglevel: int, stages_time: dict[str, float]) -> None:
+    """
+    Create debug output for execution time for each stage
+
+    :param debugdir: Output directory for the debug info.
+    :type debugdir: Directory
+    :param loglevel: Determines how detailed the output is.
+    :type loglevel: int
+    :param stages_time: Dict with execution time for each stage.
+    :type stages_time: dict
+    """
+    if loglevel < 1:
+        return
+
+    logfile = debugdir["execution_time"]
+    logtext = "".join(f"{k}: {v}\n" for k, v in stages_time.items())
+
+    with logfile.open("w") as log:
+        log.write(logtext)
+
+
+def debug_not_found_sounds(debugdir: Directory, loglevel: int, sound_name: str) -> None:
+    if loglevel < 6:
+        return
+
+    logfile = debugdir.joinpath("export/")["not_found_sounds"]
+
+    with logfile.open("a") as log:
+        log.write(sound_name+"\n")
```

Comment:
I don't know if you are using an IDE but if you do, you should configure something like `autopep8` to autoformat your code. For example, you are missing the spaces around the `+` sign here which would get auto corrected by an autoformatter.

Reply:
The `autopep8` is the recommended formatter to the openage, or can I use whatever I want?

Reply:
yes we also have `autopep8` recommended args described in 

https://github.com/SFTtech/openage/blob/master/doc/ide/vscode.md#opening-and-configuring-the-openage-workspace